architecture/apache2-node1/bootstrap.sh

```diff
@@ -72,6 +72,7 @@ mkdir -p /usr/share/modsecurity-crs/activated_rules
 #SQL Injection rules
 ln -s /usr/share/modsecurity-crs/base_rules/modsecurity_crs_41_sql_injection_attacks.conf /usr/share/modsecurity-crs/activated_rules/
 ln -s /usr/share/modsecurity-crs/base_rules/modsecurity_41_sql_injection_attacks.data /usr/share/modsecurity-crs/activated_rules/
+ln -s ./conf/xss.conf /usr/share/modsecurity-crs/activated_rules/
```

Comment:
Could you add a comment pointing out that this line enables some security rules against the XSS attacks?